pandas/tests/arrays/boolean/test_reduction.py

```diff
@@ -19,6 +19,8 @@ def data():
         ([False, pd.NA], False, False, pd.NA, False),
         ([pd.NA], False, True, pd.NA, pd.NA),
         ([], False, True, False, True),
+        ([True, True], True, True, True, True),
```

Comment:
can you add a comment with the issue number describing this case (put above these cases)

Reply:
@jreback thx for your advice. Whatsnew and comments are added.

I don't understand what to do as to add a test case in panda/tests/reductions/test_reductions.py as you described
`def test_all_any_params(self) is the relevant test`

Do you mean adding a function called `test_all_any_params` to test the completeness of parameter coverage?

Reply:
no, you look for this test where I indicated, then add a similar one (or parameterize the existing is best) with a Series that has pd.NA values. (these test whether your patch works with Series). This test you augmented here is for Array, which is fine, but need to test both.

Reply:
Thx for clarification. I add a test similar to `test_all_any_params(self)` and also test the bug exposed related to issue #33253. However, it is not a full boolean coverage test compared to the test in `pandas/tests/arrays/boolean/test_reduction.py`. Do you think it is necessary to copy the code under` @pytest.mark.parametrize` you indicate from `pandas/tests/arrays/boolean/test_reduction.py` to `panda/tests/reductions/test_reductions.py `?

Reply:
Parametrizing doesn't seem that easy, since the existing one is a object dtype with NaN and True/False, which has quite different behaviour than boolean dtype. So adding a similar but separate test as you did is fine.